instrument/instrument.go

```diff
@@ -140,6 +140,12 @@ func CollectedRequest(ctx context.Context, method string, col Collector, toStatu
 
 	if err != nil {
 		ext.Error.Set(sp, true)
+		errStr := err.Error()
+		const maxTagErrorLen = 128
```

Comment:
I'd paint it 1024. I figure we should allow tweet-length messages at a minimum.

Reply:
140? 280?

Reply:
Whichever is more presidential.